keyboards/skyloong/gk61/pro/config.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 linlin012 (@linlin012)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* Encoder Configuration */
+#ifdef ENCODER_ENABLE
+#    define ENCODER_RESOLUTION 2
+#    define ENCODER_DEFAULT_POS 0x3
+#endif
+
+/*
+ * Feature disable options
+ *  These options are also useful to firmware size reduction.
+ */
+
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+#ifdef RGB_MATRIX_ENABLE
+
+#define DRIVER_COUNT 1
+#define RGB_MATRIX_LED_COUNT 64
+#define __flash  PROGMEM
+
+/* Set Configuration for the Global Current Register */
+#define ISSI_GLOBALCURRENT 255
+
+/* Set to infinit, which is use in USB mode by default */
+//#define RGB_MATRIX_TIMEOUT RGB_MATRIX_TIMEOUT_INFINITE
+
+/* Allow shutdown of led driver to save power */
+#define RGB_MATRIX_DRIVER_SHUTDOWN_ENABLE
+/* Turn off backlight on low brightness to save power */
+#define RGB_MATRIX_BRIGHTNESS_TURN_OFF_VAL 32
+
```

Comment:
```suggestion
#define RGB_MATRIX_LED_COUNT 64
```

Reply:
Sorry, It's fail when I compile the KB. The log is as below:
--------------------------------------------------------------------------------------------------------
Compiling: keyboards/skyloong/gk61/pro/pro.c                                                       In file included from quantum/rgb_matrix/rgb_matrix.h:38,
                 from quantum/quantum.h:36,
                 from keyboards/skyloong/gk61/pro/pro.c:3:
drivers/led/issi/is31flcommon.h:46:31: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'g_is31_leds'
   46 | extern const is31_led __flash g_is31_leds[RGB_MATRIX_LED_COUNT];
      |                               ^~~~~~~~~~~
 [ERRORS]
--------------------------------------------------------------------------------------------------------
Did this means we couldn't delete "define __flash  PROGMEM" or other issue is hapened ?
Could you give me some suggestion? @waffle87

Reply:
In the absence of a better solution, let me define __flash again to solve the problem.

Reply:
Just use `PROGMEM` in the code directly — the usage of `__flash` was removed from the IS31FL3743A driver in #21343.  You need to merge or rebase your code to the current state of the `develop` branch to pick up that change.